src/content/docs/merge-queue/pause.mdx

```diff
@@ -44,6 +44,14 @@ You can also pause the merge queues programmatically using the [Mergify
 API](/api-usage). For detailed steps on how to achieve this, refer to our API
 documentation.
 
+#### Migration from the deprecated queue freeze API
+
```

Comment:
Add context:

```suggestion
The Merge Queue freeze API has been deprecated and will be removed on <date>. The feature that it provided is now covered by differents parts of Mergify, depending on your use case.

##### Freeze for a single queue rule

If you were using freeze for a partial freeze of a single queue rule (with or without cascading), you should now rely on Merge Protections to avoid pull requests entering the merge queue entirely.
(+ link)

##### Freeze for the whole queue

If you were freezing the entire merge queue, you can now leverage the Pause feature instead. It now exposes an extra argument `allow_checks_to_run` which define whether the checks should in the queue while paused (default to `false`). To mimic the behaviour of the deprecated freeze API, set this parameter to `true`.
```

You can keep/mix your content below.